parser/c2_tokenizer.c2

```diff
@@ -793,7 +787,16 @@ fn void Tokenizer.lex_internal(Tokenizer* t, Token* result) {
             result.more = false;
             return;
         case INVALID:
-            t.error(result, "invalid char '%c'", *t.cur);
```

Comment:
I like to keep the switch cases is order of the enum, so the INVALID  case should be first then

Reply:
OK, I will move it.